index.js

```diff
@@ -1,4 +1,101 @@
-//  in this file create an express application - use the middle-ware built into express
-//  to serve up static files from the public directory (index.html and client.js - you 
-//  can also serve up css files from the public directory this way if you want)
-//  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks/:name'
\ No newline at end of file
+/*
+	Scott Henderson
+	JSCRIPT 300 A Spring 2016
+*/
+
+// in this file create an express application - use the middle-ware built into express
+// to serve up static files from the public directory (index.html and client.js - you
+// can also serve up css files from the public directory this way if you want)
+// you need to support a "/trucks" endpoint, and a dynamic route for "/trucks/:name"
+
+var express = require("express");
+var _ = require("lodash");
```

Comment:
Updated package.json ... I think.  I'm pretty sure I updated the file.
Less sure I correctly created a new pull request.

On Mon, May 9, 2016 at 11:37 PM, jamdon <notifications@github.com> wrote:

> In index.js
> <https://github.com/JSCRIPT300-Spring2016/Module_4/pull/2#discussion_r62619226>
> :
>
> > \ No newline at end of file
> > +/*
> > +	Scott Henderson
> > +	JSCRIPT 300 A Spring 2016
> > +*/
> > +
> > +// in this file create an express application - use the middle-ware built into express
> > +// to serve up static files from the public directory (index.html and client.js - you
> > +// can also serve up css files from the public directory this way if you want)
> > +// you need to support a "/trucks" endpoint, and a dynamic route for "/trucks/:name"
> > +
> > +var express = require("express");
> > +var _ = require("lodash");
>
> lodash is missing from the dependencies section of your package.json file,
> meaning the server crashes when trying to run it because it can't find the
> module
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/JSCRIPT300-Spring2016/Module_4/pull/2/files/c234a5d18f9590399f0069ac9d3651e778a6e971#r62619226>
>